lib/sprite_animation.dart

```diff
@@ -50,9 +51,9 @@ class SpriteAnimation {
   /// All frames have the same [stepTime].
   SpriteAnimation.spriteList(
     List<Sprite> sprites, {
-    double stepTime,
+    @required double stepTime,
     this.loop = true,
-  }) {
+  }) : assert(stepTime != null) {
```

Comment:
Both the assert and the @required shouldn't be needed right?

Reply:
they are separate things, `@required` means you need to informa the parameter on the call, but the person could inform null, so this is why I added both

Reply:
Ah, I misread an article about @required, but of course they can't know compile time whether the reference will be null. Approved. :)